backend/src/auth/auth.controller.ts

```diff
@@ -20,17 +20,17 @@ import { AuthService } from './auth.service'
 
 @Controller('auth')
 export class AuthController {
-  constructor(
+  constructor (
     private readonly config: ConfigService,
     private readonly authService: AuthService,
     @InjectPinoLogger(AuthController.name)
-    private readonly logger: PinoLogger,
+    private readonly logger: PinoLogger
   ) {}
 
   @Post()
-  async generateOtp(
+  async generateOtp (
     @Res() res: Response,
-    @Body() generateOtpDto: GenerateOtpDto,
+      @Body() generateOtpDto: GenerateOtpDto
```

Comment:
```suggestion
     @Body() generateOtpDto: GenerateOtpDto
```

This one seems odd to format it in this manner (indenting subsequent decorators)

Reply:
See typescript-eslint/typescript-eslint#1824